docs/multicluster/architecture.md

```diff
@@ -34,64 +37,152 @@ given ClusterSet.
+
+## Multi-cluster Service
+
+### Service Export and Import
+
+<img src="assets/resource-export-import-pipeline.svg" width="1500" alt="Antrea Multi-cluster Service Export/Import Pipeline">
+
+Antrea Multi-cluster Controller implements Service export/import among member
+clusters. The above diagram depicts Antrea Multi-cluster resource export/import
+pipeline, using Service export/import as an example.
+
+Given two Services with the same name and Namespace in two member clusters -
+`foo.ns.cluster1.local` and `foo.ns.cluster2.local`, a multi-cluster Service can
+be created by the following resource export/import pipeline.
+
+* User creates a ServiceExport `foo` in Namespace `ns` in each of the two
+clusters.
+* Multi-cluster Controllers in `cluster1` and `cluster2` see ServiceExport
+`foo`, and both create two ResourceExports for the Service and Endpoints
+respectively in the Common Area of the leader cluster.
+* Multi-cluster Controller in the leader cluster sees the ResourcesExports in
+the Common Area, including the two for Service `foo`: `cluster1-ns-foo-service`,
+`cluster2-ns-foo-service`; and the two for the Endpoints:
+`cluster1-ns-foo-endpoints`, `cluster2-ns-foo-endpoints`. It then creates a
+ResourceImport `ns-foo-service` for the multi-cluster Service; and a
+ResourceImport `ns-foo-endpoints` for the Endpoints, which includes the
+exported endpoints of both `cluster1-ns-foo-endpoints` and
+`cluster2-ns-foo-endpoints`.
+* Multi-cluster Controller in each member cluster watches the ResourceImports
+from the Common Area, decapsulates them and gets Service `ns/antrea-mc-foo` and
+Endpoints `ns/antrea-mc-foo`, and creates the Service and Endpoints, as well as
+a ServiceImport `foo` in local Namespace `ns`.
+
+### Service Access Across Clusters
+
+Since Antrea v1.7.0, the Service's ClusterIP is exported as the multi-cluster
+Service's Endpoints. Multi-cluster Gateways must be configured to support
+multi-cluster Service access across member clusters, and Service CIDRs cannot
+overlap between clusters. Please refer to [Antrea Multi-cluster Gateway](#antrea-multi-cluster-gateway)
+for more information. Before Antrea v1.7.0, Pod IPs are exported as the
+multi-cluster Service's Endpoints. Pod IPs must be directly reachable across
+clusters for multi-cluster Service access, and Pod CIDRs cannot overlap between
+clusters. Antrea Multi-cluster only supports creating multi-cluster Services
+for Services of type ClusterIP.
+
+## Multi-cluster Gateway
+
+Antrea started to support Multi-cluster Gateway since v1.7.0. User can choose
+one K8s Node as the Gateway in a member cluster. The Gateway Node will be
+responsible for routing all cross-clusters traffic from the local cluster to
+other member clusters through tunnels. The diagram below depicts Antrea
+Mulit-cluster connectivity with Multi-cluster Gateways.
+
+<img src="assets/mc-gateway.svg" width="800" alt="Antrea Multi-cluster Gateway">
```

Comment:
Should we use "pod-a/b/c" to be consistent with "node-a1", etc.?

Reply:
sure, I will update it.

Reply:
@jianjuns  the change looks good to me overall, I changed a few minor descriptions and the line length for `Antrea Multi-cluster NetworkPolicy` section as well, you can move forward if you feel it's ready.